src/main/scala/org/phenoscape/kb/AnatomicalEntity.scala

```diff
@@ -19,16 +19,35 @@ import scala.concurrent.Future
 
 object AnatomicalEntity {
 
-  private val dcSource = ObjectProperty(IRI.create("http://purl.org/dc/elements/1.1/source"))
+  private val dcSource = ObjectProperty(
+    IRI.create("http://purl.org/dc/elements/1.1/source")
+  )
+
   private val ECO = IRI.create("http://purl.obolibrary.org/obo/eco.owl")
-  private val implies_presence_of_some = NamedRestrictionGenerator.getClassRelationIRI(Vocab.IMPLIES_PRESENCE_OF.getIRI)
 
-  def homologyAnnotations(term: IRI, includeSubClasses: Boolean): Future[Seq[HomologyAnnotation]] = App.executeSPARQLQueryString(homologyAnnotationQuery(term, includeSubClasses), HomologyAnnotation(_, term))
+  private val implies_presence_of_some =
+    NamedRestrictionGenerator.getClassRelationIRI(
+      Vocab.IMPLIES_PRESENCE_OF.getIRI
+    )
+
+  def homologyAnnotations(
+    term: IRI,
+    includeSubClasses: Boolean
+  ): Future[Seq[HomologyAnnotation]] =
+    App.executeSPARQLQueryString(
+      homologyAnnotationQuery(term, includeSubClasses),
+      HomologyAnnotation(_, term)
+    )
 
-  private def homologyAnnotationQuery(term: IRI, includeSubClasses: Boolean): String = {
```

Comment:
This should be fine! Which setting affects how this line is broken?

Reply:
<img width="1063" alt="Screen Shot 2020-05-11 at 2 07 00 PM" src="https://user-images.githubusercontent.com/8756596/81595672-e08afc00-9390-11ea-8e48-d5bca2d3f927.png">

Its `maxColumns` and `newlines.penalizeSingleSelectMultiArgList`. But `scalafmt` respects existing newlines and does not remove them if they don't violate the configs specified. The breaking of line here was already existing in the code as I was experimenting with configs earlier, but if I pull original code from the master branch and format it with current configs, then it shouldn't be broken up this way, since its column width is less than `maxColumns` value.

Reply:
👍